docs/howto/prepare-and-run.md

```diff
@@ -0,0 +1,70 @@
+(prepare-and-run)=
+# How to prepare and run a SQLair query
+
+A SQLair query is just like a regular SQL query, except that it contains special
+input and output expressions. These indicate to SQLair the Go objects that your
+inputs and outputs correspond to. To create and run a SQLair query there are
+several steps:
+1. Ensure the types you want to map to the query inputs and outputs are in the
+   right form for SQLair.
+2. Write your query with SQLair syntax.
+3. Prepare your query with `sqlair.Prepare`.
+4. Run your query on a database or transaction.
+
+## Prepare your Go types
+
+See {ref}`types` for how to prepare your types.
+
+## Write your query
+
+SQLair queries are regular SQL queries with special input and output expressions
+inserted which allow you to reference the Go types directly from the SQL. You
+can have any number of input and output expressions in your query.
+
+To write expressions that map Go types to query inputs see {ref}`input-expressions`.
+
+To write expressions that map Go types to query outputs see {ref}`output-expressions`.
+
+If a query contains no output expressions then SQLair will not fetch any results
+from the database. This means that you will not get a `ErrNoRows` error if you
+run a `SELECT` statement with no output expressions to check if a row exists.
```

Comment:
This sentence is hard to process. Try to make the story more linear.

Reply:
I think this actually belongs on the output expression page. Have moved it there